.tool-versions

```diff
@@ -0,0 +1,2 @@
+golang 1.16.8
+java adoptopenjdk-jre-11.0.12+7
```

Comment:
I think, we need JDK (instead of JRE) to compile Java cod.

Reply:
@asalan316 correct - its waiting in here https://github.com/cloudfoundry/app-autoscaler/pull/720 for review, but I've got no idea why these commits are in here?